src/data/users.js

```diff
@@ -6479,3 +6479,7 @@ const usersData = [
       ]
     }
   ]
+
```

Comment:
When we import this, this will need curly braces around it.

Reply:
Got it !